jsonnet/kube-prometheus/jsonnetfile.json

```diff
@@ -44,7 +44,7 @@
           "subdir": ""
         }
       },
-      "version": "master"
+      "version": "release-0.6"
```

Comment:
These we can just do as a PR to the release branch after its cut, otherwise here in master you need to undo the changes, as we always want master to point to master.

Reply:
@lilic I guess I should also do the same for kube-state-metrics?

Reply:
kube-state-metrics alerts mixin is okay to be master or pinned, either work as we did not change anything there, but we can keep it pinned to 1.9 as well. Important is to not change ksm own mixin to master, as we have breaking changes in master.